src/com/facebook/buck/ide/intellij/lang/android/AndroidLibraryModuleRule.java

```diff
@@ -53,6 +53,7 @@ public void apply(
     Optional<Path> dummyRDotJavaClassPath = moduleFactoryResolver.getDummyRDotJavaPath(target);
     if (dummyRDotJavaClassPath.isPresent()) {
       context.addExtraClassPathDependency(dummyRDotJavaClassPath.get());
```

Comment:
Can you update the description with this information? I mean the fact that Layout Preview uses compiler output, not the module dependencies.